rollup-operator/src/PoB/interface-pob.js

```diff
@@ -0,0 +1,222 @@
+const Web3 = require("web3");
```

Comment:
Definitely. 
In my opinion, it makes more sense `interface-pob` rather than `operator-manager` since it a class which basically wrapper smart contract functions. 
Then I would change `operator-manager` to `interface-pos`.